ingest-mongodb-public/src/sources/wiredTiger.ts

```diff
@@ -0,0 +1,55 @@
+import {
+  HandleHtmlPageFuncOptions,
+  handleHtmlDocument,
+  makeGitDataSource,
+} from "mongodb-rag-ingest/sources";
+
+const removeElements = (domDoc: Document) => [
+  ...Array.from(domDoc.querySelectorAll("head")),
+  ...Array.from(domDoc.querySelectorAll("script")),
+  ...Array.from(domDoc.querySelectorAll("noscript")),
+  ...Array.from(domDoc.querySelectorAll("#top")),
+  ...Array.from(domDoc.querySelectorAll(".navpath")),
+];
```

Comment:
we could, but this way is more flexible b/c it let's you select exactly what nodes to remove. like if we didn't want to do use Document.querySelectorAll(), this way would support it too. 

with the `handleHtmlDocument()` func, i tried to optimize on flexibility rather than simplicity.